pkgs/by-name/ar/archivebox/package.nix

```diff
@@ -1,104 +1,185 @@
-            "CVE-2022-28346"
-          ];
-        };
+    packageOverrides = _: super: {
+      django = super.django_5;
+      django-extensions = super.django-extensions.overridePythonAttrs {
+        # Django 5.1 compat issues
+        # https://github.com/django-extensions/django-extensions/issues/1885
+        doCheck = false;
+      };
+      bx-django-utils = super.bx-django-utils.overridePythonAttrs {
+        # Upstream has test cases for django 5.0 and 4.2 for these tests, but not 5.1, so they have to be disabled here.
+        # Note that this isn't a direct dependency, but a dependency chain.
+        # archivebox <- django-huey-monitor <- manage-django-project <- django-tools <- bx-django-utils
+        disabledTests = [
+          "test_index_page"
+          "test_basic"
+          "test_assert_html_response_snapshot"
+        ];
+      };
+      manage-django-project = super.manage-django-project.overridePythonAttrs (old: {
+        # Same reason and dependency chain as above, no Django 5.1 test snapshots so this test fails.
+        disabledTests = (old.disabledTests or [ ]) ++ [
+          "test_help"
+        ];
       });
-      django-extensions = super.django-extensions.overridePythonAttrs (old: rec {
-        version = "3.1.5";
+      django-taggit = super.django-taggit.overridePythonAttrs rec {
+        version = "6.1.0";
         src = fetchFromGitHub {
-          inherit version;
-          owner = "django-extensions";
-          repo = "django-extensions";
-          rev = "e43f383dae3a35237e42f6acfe1207a8e7e7bdf5";
-          hash = "sha256-NAMa78KhAuoJfp0Cb0Codz84sRfRQ1JhSLNYRI4GBPM=";
+          owner = "jazzband";
+          repo = "django-taggit";
+          rev = "refs/tags/${version}";
+          hash = "sha256-QLJhO517VONuf+8rrpZ6SXMP/WWymOIKfd4eyviwCsU=";
         };
-
-        # possibly a real issue, but that version is not supported anymore
-        doCheck = false;
-      });
+      };
     };
   };
 in
 
 python.pkgs.buildPythonApplication rec {
   pname = "archivebox";
-  version = "0.7.2";
+  version = "0.8.5rc53";
   pyproject = true;
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-hdBUEX2tOWN2b11w6aG3x7MP7KQTj4Rwc2w8XvABGf4=";
+  src = fetchFromGitHub {
+    owner = "ArchiveBox";
+    repo = "ArchiveBox";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-zqLutX3YM1oEgJ+Djgd8IlmnSgBRekEqX17FQqbP1Y0=";
+    # Vendors `pocket` and `pydantic-pkgr` depdendencies
+    fetchSubmodules = true;
   };
 
-  nativeBuildInputs = with python.pkgs; [
+  build-system = with python.pkgs; [
     pdm-backend
   ];
 
-  propagatedBuildInputs = with python.pkgs; [
-    croniter
-    dateparser
-    django
-    django-extensions
-    ipython
-    mypy-extensions
-    python-crontab
-    requests
-    w3lib
-    yt-dlp
+  dependencies =
+    with python.pkgs;
+    [
+      # Core Libraries
+      django
+      (django-ninja.overridePythonAttrs { doCheck = false; })
```

Comment:
Should probably comment why this is being overriden

Reply:
The checks don't pass in the nix sandbox so I had to disable them, will make a comment about it.